.eslintrc.js

```diff
@@ -17,7 +17,7 @@
 const headerBlock = [
     '******************************************************************************',
     ' *',
-    ' *    Copyright 2019 Adobe. All rights reserved.',
+    { 'pattern': ' *    Copyright 20\\d{2} Adobe. All rights reserved.' },
```

Comment:
You may want to add a `{'template':..}` entry here for people that use the "autofix" feature of VSCode